crates/ide/src/diagnostics/fixes.rs

```diff
@@ -144,6 +145,34 @@ impl DiagnosticWithFix for IncorrectCase {
     }
 }
 
+impl DiagnosticWithFix for ReplaceFilterMapNextWithFindMap {
+    fn fix(&self, sema: &Semantics<RootDatabase>) -> Option<Fix> {
+        let root = sema.db.parse_or_expand(self.file)?;
+        let next_expr = self.next_expr.to_node(&root);
+        let next_call = MethodCallExpr::cast(next_expr.syntax().clone())?;
+
+        let filter_map_call = MethodCallExpr::cast(next_call.receiver()?.syntax().clone())?;
+        let filter_map_name_range = filter_map_call.name_ref()?.ident_token()?.text_range();
+        let filter_map_args = filter_map_call.syntax().children().find_map(ArgList::cast)?;
```

Comment:
```suggestion
        let filter_map_args = filter_map_call.arg_list()?;
```
https://github.com/rust-analyzer/rust-analyzer/blob/master/crates/syntax/src/ast/traits.rs#L34-L38

Reply:
Ah! I was wondering why this method was missing - I didn't spot that it was available via a trait.